<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DishEditForm</class>
 <widget class="QWidget" name="DishEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>192</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Рецепт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteDishButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QLineEdit" name="dishNameInput">
       <property name="placeholderText">
        <string>Название блюда</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dishNameCommitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Подтвердить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="PHintLabel">
       <property name="text">
        <string>Белки:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="proteinsLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="FHintLabel">
       <property name="text">
        <string>Жиры:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fatsLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="CHintLabel">
       <property name="text">
        <string>Углеводы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="carbohsLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ккал/100 г</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="caloriesLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ingredientsLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ингредиенты:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1,1,0">
     <item>
      <widget class="QLabel" name="selectedIngredientLabel">
       <property name="text">
        <string>Выберите ингредиент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ingredientWeightInput">
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="placeholderText">
        <string>масса, г</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ingredientCommitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Подтвердить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dishIngredientDeleteButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="dishIngredientsLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
